ixmp/backend/jdbc.py

```diff
@@ -467,11 +467,14 @@ def get_data(self, ts, region, variable, unit, year):
             'value': float,
         }
 
+        fields_map = {'subannual': 'time'}
```

Comment:
renaming should happen inside the Java core, in my opinion, so that the renaming shouldn't be required in every API (ixmp, restAPI, maybe more in the future)

Reply:
it will break web API, possibly Scenario Explorer and server integration scripts and also introduce inconsistency in java code and DB model (if done without db migration)

Reply:
I'm aware that this is a major refactoring, but it is better to break the API now (before there are more active users) rather than maintaining inconsistent APIs going forward. Also, we discussed the requirement that this column be called `subannual` for weeks, so this should not come as a surprise.

Reply:
this PR contains jar returning subannual as timeseries attribute, therefore rename is removed